src/lib.rs

```diff
@@ -63,6 +63,12 @@
 //! android_logd_logger::write_event_now(1, "test").unwrap();
 //! ```
 //!
+//! To forge android logd entry:
+//!
+//! ```
+//! android_logd_logger::low(buffer_id, pid, thread_id, secs, subsec_nanos, tag, priority, message).unwrap();
```

Comment:
Here's a typo. Should be `log`.  Can you please also ensure that rustdoc catches the examples and tries to build them?